### DISCUSSION TITLE
How to run modes_in.hsd

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1447] [Q&A] [answered]
Hello everyone,

I am a new user to DFTB+ and I am following the recipe of DFTB+ to learn making calculations.

Currently I am trying to calculate the modes and I prepared the modes_in.hsd file.

Now I am submitting my job to a cluster using the following bash script:

#!/bin/bash
#SBATCH -N 1
#SBATCH --ntasks-per-node=36
#SBATCH -p cpu-long
#SBATCH -t 0-24:00:00
##SBATCH --mem=500g
#SBATCH --partition=uri-cpu,cpu,cpu-preempt
module load miniconda/22.11.1-1
conda activate dftbplus
export OMP_NUM_THREADS= $SLURM_CPUS_PER_TASK
dftb+ | tee output

Previously while running the dftb+, it used to require dftb_in.hsd but no I have modes_in.hsd file....  so The error comes saying "No dftb_in.hsd file found"

How can I specifically change my bash script so that the mode calculations will be performed?

Thank you

**Comment by aradi** [comment, marked as answer]
Make sure to call the program `modes` and **not** `dftb+` when calculating modes.
